raiden/network/proxies/token_network.py

```diff
@@ -94,13 +93,13 @@
 
 def raise_if_invalid_address_pair(address1: Address, address2: Address) -> None:
     if NULL_ADDRESS_BYTES in (address1, address2):
-        raise InvalidAddress("The null address is not allowed as a channel participant.")
+        raise ValueError("The null address is not allowed as a channel participant.")
 
     if address1 == address2:
         raise SamePeerAddress("Using the same address for both participants is forbiden.")
 
     if not (is_binary_address(address1) and is_binary_address(address2)):
-        raise InvalidAddress("Addresses must be in binary")
+        raise ValueError("Addresses must be in binary")
```

Comment:
That should be an `InvalidBinaryAddress` again, right? At least you define that exception above as 
`Raised when the address is not binary or it is not 20 bytes long.` And here the address seems not to be binary. 

... On the other hand `ValueError` and `InvalidBinaryAddress` seem not to be mutually exclusive ?!